smif/sos_model.py

```diff
@@ -4,56 +4,18 @@
 
 """
 import logging
-from enum import Enum
 
 import networkx
-import numpy as np
-from smif.intervention import Intervention, InterventionRegister
+from enum import Enum
 from smif.decision import Planning
+from smif.intervention import Intervention, InterventionRegister
 from smif.sector_model import SectorModelBuilder
 
 __author__ = "Will Usher"
 __copyright__ = "Will Usher"
 __license__ = "mit"
 
 
-class Controller:
-    """Coordinates the data-layer, decision-layer and model-runner
-
-    The Controller expects to be provided with configuration data to run a set
-    of sector models over a number of timesteps, in a given mode.
-
-    It also requires a data connection to populate model inputs and store
-    results.
-
-    Parameters
-    ----------
-    config_data : dict
-        A valid system-of-systems model configuration dictionary
-
-
-    """
-    def __init__(self, config_data):
-        builder = SosModelBuilder()
-        builder.construct(config_data)
-        self._model = builder.finish()
-
-    def run_sos_model(self):
-        """Runs the system-of-system model
-        """
-        self._model.run()
-
-    def run_sector_model(self, model_name):
-        """Runs a sector model
-
-        Parameters
-        ----------
-        model_name: str
-            The name of the sector model to run
-        """
-        self._model.run_sector_model(model_name)
-
-
 class SosModel(object):
     """Consists of the collection of timesteps and sector models
```

Comment:
Need to update the refererences to `Controller` in the docstrings to the SosModel class